docs_user/assemblies/assembly_migrating-ceph-rbd.adoc

```diff
@@ -0,0 +1,30 @@
+[id="migrating-ceph-rbd_{context}"]
+
+:context: migrating-ceph-rbd
+
+= Migrating Red Hat Ceph Storage RBD
+
+For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are running version 5 or later, you must migrate the daemons that are included in the {rhos_prev_long} control plane into the existing external RHEL nodes. The external RHEL nodes typically include the Compute nodes for an HCI environment or dedicated storage nodes.
+//kgilliga: SMEs, please check the accuracy of the second sentence.^
+To migrate Red Hat Ceph Storage Rados Block Device (RBD), your environment must meet the following requirements:
+
+* Red Hat Ceph Storage is running version 5 or later and is managed by cephadm/orchestrator.
+* Ceph NFS (ganesha) is migrated from a https://bugzilla.redhat.com/show_bug.cgi?id=2044910[TripleO based deployment to cephadm].
```

Comment:
Let's update this part to mention the Data plane adoption [1] (that bz is not relevant anymore).

[1] https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc